laws.md

```diff
@@ -86,3 +84,6 @@ This rule takes precedence over every other rule determining the winner.
 
 **301.** Ammendment of rule **201.**: Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
 
+
+**302** The state of affairs that constitutes winning may not be altered from achieving n points to any other state of affairs. The magnitude of n and the means of earning points may be changed, and rules that establish a winner when play cannot continue may be enacted and (while they are mutable) be amended or repealed.
```

Comment:
I guess. I just see it easier when the affected rule is in the rule, rather than have to see the commit history. But I get your point.